app/models/user.rb

```diff
@@ -364,6 +364,14 @@ def set_current_taxonomies
     TopbarSweeper.expire_cache(self)
   end
 
+  def my_organizations
```

Comment:
they are not, https://github.com/theforeman/foreman/pull/3732/files#r74919424 (first part)